notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.ipynb

```diff
@@ -4,42 +4,80 @@
    "cell_type": "markdown",
```

Comment:
<p>I would suggest adding titles and labels for each subplot to provide insights into what is being displayed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
Agreed, I have implemented this change.